Rakefile

```diff
@@ -0,0 +1,10 @@
+require 'rake/testtask'
+
+Rake::TestTask.new do |t|
```

Comment:
Is T a task? Perhaps choose a better iteration variable name

Reply:
use more descriptive names to help for clarity.